server/migrations/versions/2025-09-29-1146_add_recurring_interval_count_to_product_and_subscription.py

```diff
@@ -0,0 +1,51 @@
+"""Add recurring_interval_count to Product and Subscription models
+
+Revision ID: bbdc673dc236
+Revises: 902dd5c6c2b7
+Create Date: 2025-09-29 11:46:22.707113
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# Polar Custom Imports
+
+# revision identifiers, used by Alembic.
+revision = "bbdc673dc236"
+down_revision = "902dd5c6c2b7"
+branch_labels: tuple[str] | None = None
+depends_on: tuple[str] | None = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "products",
+        sa.Column("recurring_interval_count", sa.Integer(), nullable=True),
+    )
+    op.add_column(
+        "subscriptions",
+        sa.Column(
+            "recurring_interval_count",
+            sa.Integer(),
+            nullable=False,
+            server_default="1",
```

Comment:
I think we should remove the `server_default` and let us handle that on the Python side.

To make it pass during the migration, first create the column as nullable, execute the `UPDATE` below and then alter the column to make it non-nullable.